### PR TITLE
Setup and Mobile Version skeleton

### PR DESCRIPTION
**Pull request by Have-Samuel**
Setup and Mobile Version skeleton project We:
1. Made sure no Linter errors
2. Documented work professionally
3. Uesd Git & Github workflow
4. Understand how to parse a Figma design to create a UI.
5. Used Flexbox to place elements in the page.
6. Build a personal portfolio site.
7. Use images and backgrounds to enhance the look of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Version</title>
+</head>
+<body>
+  <nav>
+    <ul class="nav-bar">
+      <li class="logo">Logo</li>
+      <li><img class="hambug" src="images/Union.png" alt="hamburger"></li>
+    </ul>
+  </nav>
```
Comment (gemmen29) [inline index.html]: Kindly surround this part with a `header` tag as it helps in SEO and Accessibility

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      don't hestiate to contact me.
+    </p>
+    <ul class="items">
+      <li><img class="item" src="images/Github1.png" alt="Github"></li>
+      <li><img class="item" src="images/Inner1.png" alt="Inner"></li>
+      <li><img class="item" src="images/Peace1.png" alt="Peace"></li>
+      <li><img class="item" src="images/Twitter1.png" alt="Twitter"></li>
+      <li><img class="item" src="images/Mail1.png" alt="mail"></li>
+    </ul>
+  </section>
```
Comment (gemmen29) [inline index.html:34]: # Please match the Figma design:
| Your design | Figma design |
| --- | --- |
| image | template2 |
- The background image is overlapping with the paragraph text which makes it unreadable.
- Kindly add some spacing between elements

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Have-Samuel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @Have-Samuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**